wandb/cli/cli.py

```diff
@@ -95,6 +95,33 @@ def format_message(self):
             )
 
 
+def parse_service_config(ctx, param, value):
+    """Parse service configurations in format serviceName=policy."""
+    if not value:
+        return {}
+
+    result = {}
+    for config in value:
+        if "=" not in config:
+            raise click.BadParameter(
+                f"Service must be in format 'serviceName=policy', got '{config}'"
+            )
+
+        service_name, policy = config.split("=", 1)
+        if not service_name.strip():
```

Comment:
Instead of calling `strip()` twice, just do `service_name = service_name.strip()` after `split()`